### PR TITLE
fix chown typo

### PR DESCRIPTION
**Pull request by casz**
@crazy-max minor mistake or some chown feature I don't know about? 🤔

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ RUN apt-get update \
   && ln -sf "$JLS_PATH/bin/license-server.sh" "/usr/local/bin/license-server" \
   && groupadd -f -g 1000 jls \
   && useradd -o -s /bin/bash -d /data -u 1000 -g 1000 -m jls \
-  && chown -R jls. /data "$JLS_PATH" \
+  && chown -R jls /data "$JLS_PATH" \
```
Comment (casz) [inline Dockerfile]: or it is suppose to be
```suggestion
  && chown -R jls: /data "$JLS_PATH" \
```

why not make it explicit?
```suggestion
  && chown -R jls:jls /data "$JLS_PATH" \
```

**Comment by crazy-max** [comment]
> or some chown feature I don't know about?

Yes `jls.` = `jls:` = `jls:jls`

**Comment by casz** [comment]
😓

File: Dockerfile (hunk 1 above)
Comment (crazy-max) [reply Dockerfile]: `jls.` takes the default group

**Pull request closed without merging**